pkg/sink/sink_test.go

```diff
@@ -480,7 +510,11 @@ func Test_HandleEvent(t *testing.T) {
 		t.Fatalf("Error reading response body: %s", err)
 		return
 	}
-	wantPayload := `EventListener: my-eventlistener in Namespace: foo handling event with payload: {"head_commit": {"id": "testrevision"}, "repository": {"url": "testurl"}}`
+	wantPayload := `EventListener: my-eventlistener in Namespace: foo handling event`
```

Comment:
Can you clean this section to mirror the [original commit](https://github.com/tektoncd/triggers/commit/c9537c0636218b51766dfc5dbab4db5bdbd0389d#diff-aa07fc23750dee7511c478c5311b0d15)? We don't currently return a response.

Reply:
Actually we do.  The response is put at the end of the "HandleEvent" in sink.go.

Reply:
Ah, good point. I didn't realize we were writing to the response. Can we seed the eventid so that we can compare the body with an exact wantPayload?

Reply:
No, I can seed it.  It is generated by the eventlistener in this test.